server/adaptors/integrations/__data__/repository/haproxy/info/test/fluent-bit/fluent-bit.conf

```diff
@@ -0,0 +1,30 @@
+[SERVICE]
+    Parsers_File parsers.conf
+
+[INPUT]
+    Name forward
+    Port 24224
+
+[FILTER]
+    Name parser
+    Match haproxy.access
+    Key_Name log
+    Parser haproxy
+
+[FILTER]
+    Name    lua
+    Match   haproxy.access
+    code    function cb_filter(a,b,c)local d={}local e=os.date("!%Y-%m-%dT%H:%M:%S.000Z")d["observerTime"]=e;if c.error then d["body"]=c.remote.." "..c.frontend.." "..c.Backend..""..c.logasap.." "..c.fc_hex_c_ca_res..""..c.sq_bq..""..c.reqcookie..""..c.rescookie..""..c.terminationstate.."["..os.date("%d/%b/%Y:%H:%M:%S %z").."] \" HTTP/1.1\" "..c.code.." "..c.size.." \""..c.error.."\"" else d["body"]=c.remote.." "..c.frontend.." "..c.Backend..""..c.logasap.." "..c.fc_hex_c_ca_res..""..c.sq_bq..""..c.reqcookie..""..c.rescookie..""..c.terminationstate.."["..os.date("%d/%b/%Y:%H:%M:%S %z").."] \""..c.method.." "..c.path.." HTTP/1.1\" "..c.code.." "..c.size.." \""..c.agent.."\"" end d["trace_id"]="102981ABCD2901"d["span_id"]="abcdef1010"d["attributes"]={}d["attributes"]["data_stream"]={}d["attributes"]["data_stream"]["dataset"]="haproxy.access"d["attributes"]["data_stream"]["namespace"]="production"d["attributes"]["data_stream"]["type"]="logs"d["event"]={}d["event"]["category"]={"web"}d["event"]["name"]="access"d["event"]["domain"]="haproxy.access"d["event"]["kind"]="event"d["event"]["result"]="success"d["event"]["type"]={"access"}d["http"]={}d["http"]["request"]={}d["http"]["request"]["method"]=c.method;d["http"]["response"]={}d["http"]["response"]["bytes"]=tonumber(c.size)d["http"]["response"]["status_code"]=c.code;d["http"]["flavor"]="1.1"d["http"]["url"]=c.path;d["communication"]={}d["communication"]["source"]={}d["communication"]["source"]["address"]="127.0.0.1"d["communication"]["source"]["ip"]=c.remote;return 1,b,d end
```

Comment:
This filter looks to be generating data that isn't compatible with the provided schemas, if you set up the indices before importing data then fluentbit reports ingestion issues and no data shows up. For working with these filters better I've taken to [putting the filter in a separate file](https://gist.github.com/Swiddis/90a3767e01d4d0698436b3580adb9385), it makes debugging these issues a lot easier.

It actually looks like the issues are because this is an older version of the same filter, so the version in the Gist should work if you add the below haproxy parser and an appropriate formatter.

Reply:
Made a commit with your suggestions.